FFXIV_Modding_Tool/Validation.cs

```diff
@@ -138,9 +144,9 @@ public bool ValidateIndexFiles()
             return keepGoing;
         }
 
-        bool PromptContinuation()
+        bool PromptContinuation(message="Would you like to continue?")
```

Comment:
It is currently not specified what type of variable message is, which is required in C#. There should also be spaces before and after the = sign.

Reply:
# 🤷‍♂

Reply:
I think I did.

Reply:
Indeed, now the function is able to take an optional string.